### DISCUSSION TITLE
trouble

### DISCUSSION THREAD

**Discussion by rewandyh** [dxrk-kali/autohack#11] [General]
having trouble cloning it gets to the same point each time and says "Failed to clone 'scripts/scanning_enumeration/AutoRecon' a second time, aborting

Would you like to retry cloning ?"

this has happened everytime and i am pretty new to all of this so just looking for help

**Comment by arsic-kali** [comment]
Hey man, would you mind passing this to the issues area?

**Reply by arsic-kali** [reply]
Also, if you could copy and paste any error code or copy and paste the procedure of you trying to clone and use autohack, that would be a great help! Sorry for the late reply.
